dev-python/python-igraph/python-igraph-0.9.0.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_{7..9} )
+PYTHON_COMPAT=( python3_{7..10} )
```

Comment:
Please drop `python3_7`, it is obsolete. Did you also test it with `python3_11`? Whenever possible it is nice to add both 10 and 11 at the same time to save time later.